credentials/src/main/kotlin/web5/sdk/credentials/model/Validator.kt

```diff
@@ -0,0 +1,125 @@
+package web5.sdk.credentials.model
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.nfeld.jsonpathkt.JsonPath
+import com.nfeld.jsonpathkt.extension.read
+import web5.sdk.credentials.JsonPathParseException
+
+/**
+ * PresentationDefinitionV2Validator Validator.
+ **/
+public object PresentationDefinitionV2Validator {
+
+  /**
+   * Validates a PresentationDefinitionV2.
+   */
```

Comment:
Here and elsewhere, I'd suggest adding which validations are done. Either explicitly, or pointing to the spec.